docs/azure_arc_jumpstart/azure_arc_k8s/developer/dapr/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+type: docs
+title: "Dapr"
+linkTitle: "Dapr"
```

Comment:
Might be good to be more explicit here and mention the name of the scenario in the context of Dapr. i.e Dapr state store integration with Redis" or something like tthat